README.md

```diff
@@ -12,108 +12,111 @@
+Successfully created project my-starter-project.
 
-Once the deployment has completed, you can reach your new service by going to https://app.localhost.architect.sh.
+Your project is ready to be deployed by Architect!
+To deploy locally, run:
+  architect dev my-starter-project/architect.yml
+```
 
 ```sh
-# Clone the repository and navigate to this directory
-$ git clone git@github.com:architect-templates/nuxt.git
+$ architect init nuxt
 $ cd ./nuxt
-
-# Deploy locally using the dev command
-$ architect dev architect.yml
 ```
 
-## Deploying to the Cloud
-
-Want to try deploying this to a cloud environment? Architect's got you covered there, too! It only takes a minute to [sign up for a free account](https://cloud.architect.io/signup).
-
-You can then [deploy the application](https://docs.architect.io/getting-started/introduction/#deploy-to-the-cloud) by running the command below. Note that “example-environment” is the free environment that is created with your Architect account.
+### Clone it yourself
+Run the following command to clone the repo yourself:
 
 ```sh
-# Deploy to Architect Cloud
-$ architect deploy architect.yml -e example-environment
+# Clone the repository and navigate to this directory
+$ git clone git@github.com:architect-templates/nuxt.git
+$ cd ./nuxt
 ```
 
----
-# Additional Learning
-After you feel comfortable deploying with Architect, check out the powerful features we've added into this component!
+## Run Locally
+Once the repo has been cloned to your local machine, execute the following command from the `nuxt` directory to run it locally:
 
-Additional features within this component:
-* [Adding a dependency](#adding-a-dependency)
+```sh
+$ architect dev .
+```
 
+When this command completes, you can reach your new application at https://app.localhost.architect.sh.
+### Make your own changes
 
-# Adding a dependency
-Using a microservice framework allows you to decouple an application and alleviate many of the problems that come with a monolithic architecture. Architect provides first class support for microservces through the use of dependencies.
+This application's `architect.yml` file contains a `debug` block that enables hot-reloading for each service
+within the component. That means you can make changes to the source code, which will automatically apply to the environment. This lets you quickly iterate and see your changes without restarting the
+application stack.
 
-This example application is configured to connect to the [Node.js Starter Project](https://github.com/architect-templates/node-rest-api) REST API as a [dependency](https://docs.architect.io/components/dependencies/), but you can modify the `architect.yml` file to connect to any REST API backend as an Architect Component.
+Give it a try! Search inside your project for "Favorite Movie” and change this string to “Favorite Pizza.” Once you saved
```

Comment:
This string doesn't exist on the page

![image](https://user-images.githubusercontent.com/8558567/226747227-fc0f6306-e782-4b9b-aae4-bcf40b65e207.png)

Reply:
Fixed
![image](https://user-images.githubusercontent.com/532523/226748914-cad43c91-04fe-4736-9b90-70018d715727.png)